web/src/components/navbar/NavBar.test.tsx

```diff
@@ -94,6 +94,30 @@ describe("<NavBar />", () => {
     useMockRouter({});
   });
 
+  it("shows business name when showSidebar is true", () => {
+    useMockUserData(generateBusinessNamedUserData());
+    setLargeScreen(false);
+    render(
+      withAuth(<NavBar landingPage={false} task={undefined} showSidebar={true} />, {
+        isAuthenticated: IsAuthenticated.TRUE,
+      })
+    );
+
+    expect(screen.getByText(businessName)).toBeInTheDocument();
+  });
+
+  it("does not show business name when showSidebar is false", () => {
+    useMockUserData(generateBusinessNamedUserData());
+    setLargeScreen(false);
+    render(
+      withAuth(<NavBar landingPage={false} task={undefined} showSidebar={false} />, {
+        isAuthenticated: IsAuthenticated.TRUE,
+      })
+    );
+
+    expect(screen.queryByText(businessName)).not.toBeInTheDocument();
+  });
+
```

Comment:
Is it possible to add a test for the truncation behavior?

Reply:
Not meaningfully, I can check for the class or the style attributes but the way that jest renders tests its a css visual change not an underlying element change. Does that make sense?

Reply:
Yeah that makes sense